src/iv/imageviewer.cpp

```diff
@@ -64,7 +64,7 @@ IsSpecSrgb(const ImageSpec& spec)
 static const char *s_file_filters = ""
     "Image Files (*.bmp *.cin *.dcm *.dds *.dpx *.fits *.gif *.hdr *.ico *.iff "
     "*.jpg *.jpe *.jpeg *.jif *.jfif *.jfi *.jp2 *.j2k *.jxl *.exr *.png *.pbm *.pgm "
-    "*.ppm *.psd *.ptex *.rla *.sgi *.rgb *.rgba *.bw *.int *.inta *.pic *.tga "
+    "*.ppm *.psd *.ptex *.R3D *.r3d *.rla *.sgi *.rgb *.rgba *.bw *.int *.inta *.pic *.tga "
```

Comment:
I don't think you need the capitals -- I believe it's a case-insensitive compare? Otherwise, all of the above would need them.

Reply:
I added capitals after failure in test run of the image viewer. Will dig deeper and check.

Reply:
https://doc.qt.io/qt-6/qfiledialog.html#getSaveFileName doesn't say one way or the other if the filters file file extensions are case sensitive or not, but the examples in the docs are all lower case only. I would assume that if they were case sensitive, their examples would include the upper case versions, too, or they would get endless complaints from Windows users?

Reply:
OK, leave both capitalizations, then.